crowbar_framework/index/index.html

```diff
@@ -115,6 +115,7 @@
 
     <script>
       document.getElementById("dashboard").href="http://" + window.location.host + ":3000";
+      window.location.replace(document.getElementById("dashboard"));
```

Comment:
(not that it matters much, but I'd use a temporary variable to store the URL and use it on the two lines)